api_backend.yaml

```diff
@@ -307,6 +307,35 @@ paths:
           description: Profile cannot be updated.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/user-identity":
+    x-swagger-router-controller: ProfileController
+    get:
+      operationId: getSpidProfile
```

Comment:
We are going to add new authentication methods so i think is better to be generic (no hardcoded SPID).

@cc @cloudify

Reply:
agreed

Reply:
A more generic `getUserIdentity` could be better?

Reply:
Fixed in 2e6a1bf

Reply:
it looks like it's not commited yet

Reply:
Fixed in 4fd9645